resource_packs/files/peace_and_quiet/quieter_water/sounds.json

```
@@ -27,6 +27,9 @@
 			},
 			"cauldron_drip.water.pointed_dripstone": {
 				"volume": 0.1
+			},
+			"drip.water.pointed_dripstone" : {
+			   "volume" : 0.1
```

```suggestion
			   "volume" : [0.015, 0.05]
```
Do this so that it's actually reducing 95% from vanilla and random selection of volume will be selected when the sound is played because that's how vanilla do **(make sure to format the code :D)**